micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/Configuration.java

```diff
@@ -162,6 +162,215 @@ public void setProperties(@MapFormat(transformation = MapFormat.MapTransformatio
          */
         // Close #219 - This method is needed so that the auto-completion of properties works
         boolean isIndexRedirectEnabled();
+
+        @NotNull
+        HeaderSecurity getHeadersecurity();
+
+        @ConfigurationProperties("header-security")
```

Comment:
Do we need to explicitly set the name to "header-security?" I think it should work without it because kebab case and camel case are both supported by default.